app/Http/Livewire/CreateBatch.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Http\Livewire;
+
+use Livewire\Component;
+
+class CreateBatch extends Component
```

Comment:
Kindly delete this unused livewire component

Reply:
Deleted the unussed livewire component.